spec/controllers/year_select_controller_spec.rb

```diff
@@ -12,14 +12,10 @@
 
   describe "#update" do
     context "with a valid year" do
-      it "redirects to the 2023 path" do
+      it "redirects to the edit email address path and saves the year to session" do
         post :update, params: { year_select_form: { year: "2023" } }
-        expect(response).to redirect_to(root_path)
-      end
-
-      it "redirects to the 2024 path" do
-        post :update, params: { year_select_form: { year: "2024" } }
-        expect(response).to redirect_to(root_path)
+        expect(response).to redirect_to(edit_email_address_path)
+        expect(session[:year_selected]).to eq("2023")
```

Comment:
shouldn’t this be 2024?

Reply:
why?

Reply:
i got lost by what was changed and what was here. please disregard.

Reply:
but, it might be worth testing the different valid years? idk if that will take long to change/add.